enhancements/rhcos/marketplace-images.md

```diff
@@ -0,0 +1,470 @@
+OpenShift relesae cycle. After reviewing [the commit history for changes to the RHCOS stream](https://github.com/openshift/installer/commits/main/data/data/coreos),
+we see that there were 10 bumps to rhcos.json in 2024, but several of these bumps were pre-release. To mitigate unnecessary toil,
+we propose that ARO would only be expected to bump marketplace images on release branches.
+
+### Drawbacks
+
+One drawback is that this potentially increases the operational burden on the installer team to take on responsibility for
+boot images or at least to facilitate the update process. That said, this burden does not seem significant and something that
+could be addressed in the future if needed.
+
+## Open Questions [optional]
+
+* What should be the process for notifying publishers that RHCOS has been bumped and we need new images?
+** Jira ticket: Process wise, this is ideal, but there is not github/prow automation to create Jira tickets.
+** Slack alert: it is possible to integrate slack alerts with prow postsubmits (so an alert is generated when the PR is merged)
+** Test failure: we plan on introducing tests to inform when there is version skew (see below), so these could be used as well 
+
+* Could the RHCOS or an upstream team take on publishing SCOS to an Azure community gallery and including it in the scos stream?
+It seems like this be added as part of the standard stream--not an extension like marketplace images--and as a result it would need
+to be handled by the team responsible for bumping SCOS images. The Installer team would be willing to take on some of this toil, if needed.
+**Answered: Maybe** We will defer this (see graduation criteria below). FCOS is making progress in 
+https://github.com/coreos/fedora-coreos-tracker/issues/148.
+
+* Should Azure GovCloud, and therefore other Azure clouds get treatment in the RHCOS stream? The marketplace is separate for different clouds
+and ARO publishes unpaid marketplace images in Azure Government cloud. My opinion is that we should depend on consistent publishing across
+clouds to reduce cardinality within the stream; that is, rather than having top-level fields for `AzureCloud` and `AzureGovernment`, we should
+ensure that images are published with the same details in both clouds. **Answered: No** There is no. distinction in the publication of images to
+MAG, so we do not need this distinction.
+
+* Now that the RHCOS stream has moved to versioning releases based on the RHEL version, can we standardize the way the image versions are listed
+in the paid marketplace and ARO? For example, the most recent rhcos release in the installer is [. ](https://github.com/openshift/installer/blob/fc9c0c69f117ac423d2e83fd7c3c94ae63924934/data/data/coreos/rhcos.json#L11C23-L11C37), compared to `418.94.202501221327-0` for 4.18. Ideally, both ARO and paid marketplace images would use the exact
+release as the version, trimming the suffix of the patch version as needed, according to Azure requirements.
```

Comment:
@bennerv FYI, I just added this open question about the new version formats for rhcos